### PR TITLE
Update OpenTherm Gateway docs to reflect the rewrite to a component.

### PR DESCRIPTION
**Pull request by mvn23**
**Description:**
OpenTherm Gateway has been rewritten to a component. This PR updates the docs accordingly

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17133

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/climate.opentherm_gw.markdown [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: page
-title: "OpenTherm Gateway"
+title: "OpenTherm Gateway Climate"
 description: "Control your OpenTherm Gateway from Home Assistant."
-date: 2018-08-29 16:23
+date: 2018-10-09 11:23
```
Comment (frenck) [inline source/_components/climate.opentherm_gw.markdown]: Please do not update these dates.
The date indicates the initial introduction of the file.

File: source/_components/opentherm_gw.markdown [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+title: "OpenTherm Gateway Hub"
+description: "Control your OpenTherm Gateway from Home Assistant."
+date: 2018-10-07 16:23
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: opentherm.png
+ha_category: Hub
+ha_release: "0.80"
```
Comment (frenck) [inline source/_components/opentherm_gw.markdown]: This is going to be `0.81`.

File: source/_components/opentherm_gw.markdown [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      default: "OpenTherm Gateway"
+    precision:
+      description: "The desired precision for this device. Can be used to match your actual thermostat's precision. Supported values are `0.1`, `0.5` and `1.0`."
+      required: false
+      type: float
+      default: "`0.5` for Celsius and `1.0` for Fahrenheit."
+    floor_temperature:
+      description: "Some thermostats round all temperatures down to the lower value according to their precision. Default behaviour for Home Assistant is to round temperatures to the nearest value. Set this to `True` to override Home Assistant and round to the lower value according to the configured `precision`."
+      required: false
+      type: boolean
```
Comment (frenck) [inline source/_components/opentherm_gw.markdown:50]: Type `boolean` should have a default.

File: source/_components/opentherm_gw.markdown [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+      required: false
+      type: string
+      default: "OpenTherm Gateway"
+    precision:
+      description: "The desired precision for this device. Can be used to match your actual thermostat's precision. Supported values are `0.1`, `0.5` and `1.0`."
+      required: false
+      type: float
+      default: "`0.5` for Celsius and `1.0` for Fahrenheit."
+    floor_temperature:
+      description: "Some thermostats round all temperatures down to the lower value according to their precision. Default behaviour for Home Assistant is to round temperatures to the nearest value. Set this to `True` to override Home Assistant and round to the lower value according to the configured `precision`."
```
Comment (frenck) [inline source/_components/opentherm_gw.markdown]: Our docs are American English:
- `behaviour` -> `behavior`

**Review by frenck — CHANGES_REQUESTED** [review]
@mvn23 Thanks for the PR!
I've reviewed it and left you some comments. Could you please take a look? 👍

**Comment by mvn23** [comment]
Thanks for the review, all requested changes have been made and your comments have been noted.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks for processing those @mvn23! 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**